extensions/wikia/AdEngine/js/video/player/porvata/googleImaPlayerFactory.js

```diff
@@ -33,6 +33,14 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory', [
 			}
 		}
 
+		function setAutoplay(value) {
+			// videoAd DOM element is present on mobile only
+			if (videoAd) {
+				videoAd.autoplay = value;
+				videoAd.muted = value;
```

Comment:
nitpick: fact that while setting autoplay we are also setting `muted` property can be counterintuitive